website/docs/execution-node/fee-recipient.md

```diff
@@ -1,83 +1,172 @@
 ---
 id: fee-recipient
-title: Setting an Address for Receiving Transaction Fees
+title: How to configure Fee Recipient on your client instance and/or beacon node
 sidebar_label: Setting an Address for Receiving Transaction Fees
+doc_type: how-to
+sme: james-prysm
 ---
 
-# Fee Recipient
 
-Validators don't currently receive any portion of the transaction fees paid by end-users when proposing new blocks.
+# How to configure Fee Recipient on your client instance and/or beacon node
```

Comment:
Updating # H1 to make it clear to the reader that they're reading a how-to with a specific goal in mind, not a concept.

Reply:
Reverting based on how clumsy it feels when rendered in large font, and based on the current pattern being followed in this node's sidebar.